extending-valtimo/creating-data-provider.md

```diff
@@ -0,0 +1,65 @@
+# What is a data provider
+
+A data provider is a simple way of exposing any kind of data through an API. An engineer can expose the data by writing
+a data provider. 
+
+The data provider setup was created for a specific use-case: the admin of the application should be able to configure a
+data source that should populate a dropdown list. The admin can pick any data provider from the ui, without any code
```

Comment:
ui should either be UI (capitalized), or just write it without the abbreviation.